Sources/SearchableCollectionViewController/SearchableCollectionController.swift

```diff
@@ -62,17 +59,7 @@ open class SearchableCollectionController: SweetCollectionController {
     }
 }
 
-extension SearchableCollectionController: UICollectionViewDataSource {
-    open func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        return UICollectionViewCell()
-    }
-
-    open func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return 0
-    }
-}
-
-extension SearchableCollectionController: UICollectionViewDelegate {
+extension SearchableCollectionController: UIScrollViewDelegate {
     open func scrollViewDidScroll(_ scrollView: UIScrollView) {
```

Comment:
Yeah, maybe assuming that every user will do

```swift
self.collectionView.delegate = self
```

on their subclasses might be a bit too much, having our helper implement the delegate doesn't feel as clean but I understand why that would be convenient. I rolled back that change.